mlir/test/Target/LLVMIR/Import/call-attributes.ll

```diff
@@ -0,0 +1,18 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
```

Comment:
```suggestion
; RUN: mlir-translate -import-llvm %s | FileCheck %s
```
nit: the split is not needed here, however it could make sense to move the test to call-argument-attributes.ll and add a split?